backend/build.gradle.kts

```diff
@@ -4,10 +4,10 @@ import org.springframework.boot.gradle.tasks.run.BootRun
 plugins {
     id("org.springframework.boot") version "3.1.2"
     id("io.spring.dependency-management") version "1.0.15.RELEASE"
-    kotlin("jvm") version "1.9.10"
-    kotlin("plugin.spring") version "1.9.10"
-    kotlin("plugin.jpa") version "1.9.10"
-    kotlin("kapt") version "1.9.10" // needed for query-dsl
```

Comment:
I tried to run the backend using version 1.9.10 but kept getting conflict errors on my machine. Therefore, I had to upgrade it to develop locally. However, I can revert it back to version 1.9.10 if you prefer using that on your end!